core/src/services/lakefs/core.rs

```diff
@@ -143,6 +143,38 @@ impl LakefsCore {
 
         self.client.send(req).await
     }
+
+    pub async fn upload_object(
```

Comment:
Hi, we don't need to build a request and send it later; we can send it directly here.

Reply:
I see.